ComposureWatch/src/Components/PlayerCards.jsx

```diff
@@ -1,6 +1,160 @@
+    },
+    {
+      name: "Doomfist",
+      img: "src/IMG/portraits/doomfist.png",
+    },
+    {
+      name: "JunkerQueen",
+      img: "src/IMG/portraits/junkerQueen.png",
+    },
+    {
+      name: "Orisa",
+      img: "src/IMG/portraits/orisa.png",
+    },
+    {
+      name: "Ramattra",
+      img: "src/IMG/portraits/ramattra.png",
+    },
+    {
+      name: "Reinhardt",
+      img: "src/IMG/portraits/reinhardt.png",
+    },
+    {
+      name: "Roadhog",
+      img: "src/IMG/portraits/roadHog.png",
+    },
+    {
+      name: "Sigma",
+      img: "src/IMG/portraits/sigma.png",
+    },
+    {
+      name: "Winston",
+      img: "src/IMG/portraits/winston.png",
+    },
+    {
+      name: "Wrecking Ball",
+      img: "src/IMG/portraits/wreckingBall.png",
+    },
+    {
+      name: "Zarya",
+      img: "src/IMG/portraits/zarya.png",
+    },
+    {
+      name: "Ashe",
+      img: "src/IMG/portraits/ashe.png",
+    },
+    {
+      name: "Bastion",
+      img: "src/IMG/portraits/bastion.png",
+    },
+    {
+      name: "Cassidy",
+      img: "src/IMG/portraits/cassidy.png",
+    },
+    {
+      name: "Echo",
+      img: "src/IMG/portraits/echo.png",
+    },
+    {
+      name: "Genji",
+      img: "src/IMG/portraits/genji.png",
+    },
+    {
+      name: "Hanzo",
+      img: "src/IMG/portraits/hanzo.png",
+    },
+    {
+      name: "Junkrat",
+      img: "src/IMG/portraits/junkrat.png",
+    },
+    {
+      name: "Mei",
+      img: "src/IMG/portraits/mei.png",
+    },
+    {
+      name: "Pharah",
+      img: "src/IMG/portraits/pharah.png",
+    },
+    {
+      name: "Reaper",
+      img: "src/IMG/portraits/reaper.png",
+    },
+    {
+      name: "Sojourn",
+      img: "src/IMG/portraits/sojourn.png",
+    },
+    {
+      name: "Soldier76",
+      img: "src/IMG/portraits/soldier76.png",
+    },
+    {
+      name: "Sombra",
+      img: "src/IMG/portraits/sombra.png",
+    },
+    {
+      name: "Symmetra",
+      img: "src/IMG/portraits/symmetra.png",
+    },
+    {
+      name: "Torbjorn",
+      img: "src/IMG/portraits/torbjorn.png",
+    },
+    {
+      name: "Tracer",
+      img: "src/IMG/portraits/tracer.png",
+    },
+    {
+      name: "Widowmaker",
+      img: "src/IMG/portraits/widowmaker.png",
+    },
+    {
+      name: "Ana",
+      img: "src/IMG/portraits/ana.png",
+    },
+    {
+      name: "Baptiste",
+      img: "src/IMG/portraits/baptiste.png",
+    },
+    {
+      name: "Brigitte",
+      img: "src/IMG/portraits/brigitte.png",
+    },
+    {
+      name: "Kiriko",
+      img: "src/IMG/portraits/kiriko.png",
+    },
+    {
+      name: "Lucio",
+      img: "src/IMG/portraits/lucio.png",
+    },
+    {
+      name: "Mercy",
+      img: "src/IMG/portraits/mercy.png",
+    },
+    {
+      name: "Moira",
+      img: "src/IMG/portraits/moira.png",
+    },
+    {
+      name: "Zen",
+      img: "src/IMG/portraits/zen.png",
+    },
+  ];
+
+  let charImg;
+  if (props.character == "Moira") {
```

Comment:
I would recommend using `===` instead of `==` ...

> "Strict equality is almost always the correct comparison operation to use." - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness